web/embed-html.html

```diff
@@ -33,19 +33,36 @@
           rel="stylesheet" media="screen">
     <link rel="import" href="embed_components.html">
     <script type="application/dart" src="embed.dart"></script>
+    <style>
+      #run-button {
+        background-color: #d23f31;
+      }
+
+      .button-bar {
```

Comment:
OK, but these styles should then be moved into the `<dom-module id="dartpad-buttons">` custom component - we shouldn't have copies in the different embedder files.